syntax = "proto3";

package packet;

message CountDownPacket {
	float interval = 1;
}